packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -249,23 +249,29 @@ export default class BackupNg {
             }),
           ])
 
-          // Fails the job if all remotes are disabled
+          // update remotes list with only the enabled remotes
+          // only keep the destination remote in case of a mirror backup
+          job.remotes = {
+            id: {
+              __or: Object.keys(remotes).filter(remoteId => remoteId !== job.sourceRemote),
+            },
+          }
+
+          // Fails the job if all the target remotes are disabled
           //
           // TODO: integrate each failure in its own tasks and still proceed
           // with other tasks like rolling snapshot and replication.
-          if (remoteIds.length > 0 && Object.keys(remotes).length === 0) {
+          if (remoteIds.length > 0 && Object.keys(job.remotes).length === 0) {
```

Comment:
This test is incorrect, what it should be testing: if there are target remotes and if all of them are disabled.

It was changed previously in https://github.com/vatesfr/xen-orchestra/pull/6437/files